src/AppInstallerCLICore/COMContext.h

```diff
@@ -40,14 +40,12 @@ namespace AppInstaller::CLI::Execution
         {
             Reporter.SetChannel(Reporter::Channel::Disabled);
             Reporter.SetProgressSink(this);
-            SetFlags(CLI::Execution::ContextFlag::AgreementsAcceptedByCaller);

```

Comment:
ContextFlag::AgreementsAcceptedByCaller is specifically created for ComContext, since ComContext is no longer using it, this falg definition should be removed from Context

Reply:
Removed Agreementsacceptedbycaller flag entirely.